types/prosemirror-commands/index.d.ts

```diff
@@ -4,13 +4,26 @@
 //                 David Hahn <https://github.com/davidka>
 //                 Tim Baumann <https://github.com/timjb>
 //                 Patrick Simmelbauer <https://github.com/patsimm>
+//                 Mike Morearty <https://github.com/mmorearty>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.0
 
 import { MarkType, Node as ProsemirrorNode, NodeType, Schema } from 'prosemirror-model';
 import { EditorState, Transaction } from 'prosemirror-state';
 import { EditorView } from 'prosemirror-view';
 
+export interface Command<S extends Schema = any> {
+  (
+    state: EditorState<S>,
+    dispatch: (tr: Transaction<S>) => void,
+    view: EditorView<S>
+  ): boolean;
+}
+
+export interface Keymap<S extends Schema = any> {
```

Comment:
Are there any reasons for using `any` instead of `unknown` here? If not, could you please replace `any` with `unknown`? (I believe that this would also require using the type `Keymap<any>` for `baseKeymap`, `pcBaseKeymap` and `macBaseKeymap` because otherwise using one of these values would not be possible when something of type `Keymap<S>` for some concrete schema type `S` is required.)

Reply:
(see other comment)